docker-compose.yml

```diff
@@ -12,30 +13,99 @@ services:
       - "5432:5432"
     volumes:
       - postgresql_data:/var/lib/postgresql/data
-  service:
+  service-without-db:
```

Comment:
Could you rename this to something like `base-service` or the like? It's clear here, but quite confusing where it's use in other services (I.e., the ones that extend this, but depend on `db` feel somehow confused).